v2/tools/generator/internal/codegen/pipeline/add_cross_resource_references_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/config"
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/test"
+)
+
+var IDsArrayProperty = astmodel.NewPropertyDefinition("Ids", "ids", astmodel.NewArrayType(astmodel.ARMIDType))
```

Comment:
Suggest adding a comment to this property, so the golden file points out the important thing:

```suggestion
var IDsArrayProperty = astmodel.NewPropertyDefinition("Ids", "ids", astmodel.NewArrayType(astmodel.ARMIDType)).
    WithComment("This is the important line here. The final type should be genruntime.ResourceReference")
```

Reply:
Done